docs/03_server/10_integration/12_open-api.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Open API'
+sidebar_label: 'Open API'
+id: open-api
+keywords: [server, integration, open api, introduction]
+tags:
+  - server
+  - integration
+  - open api
+  - introduction
+---
+
+:::important
+
+Open API support was introduced in version 7.0 of the Genesis platform.
+
+:::
+
+The Genesis low-code platform supports the use of [Open API](https://swagger.io/specification/) in order to integrate with external systems. 
+
+Genesis makes this easy to configure and set up, and an Open API specification is generated for the following resources:
+
+* [Request Server](../../../../server/request-server/introduction/)
+  * The specification is generated for every GPAL request server.
+* [Event Handler](../../../../server/event-handler/introduction/)
+  * The specification is generated for each GPAL event handler.
+* [Custom Endpoints](../custom-endpoints/introduction/)
+  * The specification is generated for each GPAL Custom Endpoints.
+  * The standard specification can be augmented using the `openapi { ... }` DSL, [see here for details](../custom-endpoints/advanced/#openapi).
+
+## Accessing Open API specification
+
+:::important
+
+Genesis Open API supports Open API version 3.1.
+
+:::
+
+The Open API specification is generated at runtime and is available at the following endpoints:
+
+Full specification: 
+`https://{server-host}/gwf/open-api/spec/`
+
+Downloadable specification: 
+`https://{server-host}/gwf/open-api/spec.yaml`
+
+Service specific specification: 
+`https://{server-host}/gwf/open-api/spec/{service-name}`
```

Comment:
Maybe an example of service-name or how to get a list of service names ?

Reply:
you would need to check the server